hail/python/hail/docs/change_log.md

```diff
@@ -24,6 +24,28 @@ an earlier version of Hail to read files written in a later version.
 
 ---
 
+## Version 0.2.29
+
+Released 2019-12-17
+
+### Bug fixes
+- (hail#7229) Fixed `hl.maximal_independent_set` tie breaker functionality.
+- (hail#7732) Fixed incompatibility with old files leading to incorrect data read when filtering intervals after `read_matrix_table`.
+- (hail#7642) Fixed crash when constant-folding functions that throw errors.
+- (hail#7611) Fixed `hl.hadoop_ls` to handle glob patterns correctly.
+- (hail#7653) Fixed crash in `ld_prune` by unfiltering missing GTs. 
+
+### Performance improvements
+- (hail#7719) Generate more efficient IR for `Table.flatten`
```

Comment:
period at end (minor nit, but you'll need to edit for the horizontal bar anyway)